gammapy/maps/region/geom.py

```diff
@@ -683,16 +697,22 @@ def from_hdulist(cls, hdulist, format="ogip", hdu=None):
             region_hdu = hdu + "_" + region_hdu
 
         if region_hdu in hdulist:
-            region_table = Table.read(hdulist[region_hdu])
-            wcs = WcsGeom.from_header(region_table.meta).wcs
+            try:
```

Comment:
Maybe add a reminder TODO to remove the `try` once regions 0.5 is no longer supported

Reply:
OK